ChangeLog.d/sha256-armce-arm.txt

```diff
@@ -0,0 +1,4 @@
+Features
+    * Support Armv8 Crypto Extension acceleration for SHA-256
+      when compiling for Thumb or 32-bit Arm.
+
```

Comment:
(a) extra blank line at the end of the file
(b) PR title references AES, this is about SHA-256 - isn't that #8298?

Reply:
Again, you've commented on the wrong PR :-)

Reply:
Is this PR missing `needs-preceding-pr`?

Reply:
It's based on 8298 in order to pick up the definition of `MBEDTLS_IS_ARMV8`. Yes, I'll add the label.